packages/frontend/src/hooks/thirdPartyServices/useSentry.ts

```diff
@@ -32,6 +44,7 @@ const useSentry = (
                     return sentryConfig.tracesSampleRate;
                 },
                 replaysOnErrorSampleRate: 1.0,
+                debug: true,
```

Comment:
should this be true always or depending on ENV?

Reply:
🤦 completely missed this, removing